### DISCUSSION TITLE
Include support for Pushover API in History plugin

### DISCUSSION THREAD

**Discussion by steve51184** [Novik/ruTorrent#2182] [Ideas]
I see the history plugin supports Pushbullet notifications but that is quite limited in usage (500 messages a month via API) where as services like Pushover have a limit of 10,000 API messages

Those 500 from pushbullet really don't last long if you use pushbullet for other things.. Pushover would be a nice edition to the plugin for heavier users

**Comment by pinkairfreshener** [comment]
Would definitely love this addition!
